rgf/sklearn.py

```diff
@@ -624,6 +629,14 @@ def fit(self, X, y, sample_weight=None):
                 self._classes_map[i] = cls_num
                 ovr_list[i] = (y == cls_num).astype(int)
                 self._estimators[i] = _RGFBinaryClassifier(**params)
+
+            n_jobs = self.n_jobs if self.n_jobs > 0 else cpu_count() + self.n_jobs + 1
+            substantial_njobs = n_jobs if n_jobs <= self.n_classes_ else self.n_classes_
```

Comment:
You could use `max(n_jobs, self.n_classes_)` here. And what about the special case when `classes_ = 2`?

Reply:
Message for the case when `classes_ == 2` is written in l.621.

Reply:
Yeah, exactly! Sorry for false alarm 😄 .